testFixtures/v3.1-RC/manifest.json

```diff
@@ -0,0 +1,156 @@
+{
+    "last_updated": "2019-07-04T13:33:03.969Z",
+    "ttl": 60,
+    "version": "3.1-RC",
+    "data": {
+        "datasets": [
+            {
+                "system_id": "tier_fingal",
+                "versions": [
+                    {
+                        "url": "https://platform-services.tier-services.io/data-sharing/tier_tier_fingal/gbfs/2.1",
+                        "version": "2.1"
+                    },
+                    {
+                        "url": "https://platform-services.tier-services.io/data-sharing/tier_tier_fingal/gbfs/2.2",
+                        "version": "2.2"
+                    },
+                    {
+                        "url": "https://platform-services.tier-services.io/data-sharing/tier_tier_fingal/gbfs/2.3",
+                        "version": "2.3"
+                    },
+                    {
+                        "url": "https://platform-services.tier-services.io/data-sharing/tier_tier_fingal/gbfs/3.1-RC",
+                        "version": "3.1-RC"
+                    }
+                ]
```

Comment:
Suggestion to add `area` and `country_code` in the manifest.json test to cover https://github.com/MobilityData/gbfs/pull/572. 

Example was found in https://github.com/MobilityData/gbfs/blob/master/gbfs.md#manifestjson.

```suggestion
                ],
                "area": {
                  "type": "MultiPolygon",
                  "coordinates": [
                    [
                      [
                        [
                          13.10821,
                          52.58563
                        ],
                        [
                          13.29743,
                          52.67046
                        ],
                        [
                          13.48451,
                          52.6855
                        ],
                        [
                          13.77993,
                          52.43458
                        ],
                        [
                          13.65355,
                          52.33048
                        ],
                        [
                          13.08165,
                          52.38793
                        ],
                        [
                          13.10821,
                          52.58563
                        ]
                      ]
                    ]
                  ]
                },
                "country_code": "DE"
```